### ISSUE TITLE
Accessibility : Need alternative text for images.

### ISSUE THREAD

**Issue by sahilbucha** [swagger-api/swagger-ui#5800]
**Q&A (please complete the following information)**
OS: Linux
Browser: Chrome
Swagger-UI version: 3.0.7

**Content & configuration**
This violation is referring to the SVGs used for the unlocked and locked symbols used before and after a user has entered authorization details into the authorization dialog.

**Describe the bug you're encountering**
The SVGs used throughout the page are described in their button element and without proper attributes may be misread.

<svg width="20" height="20"><use href="#unlocked" xlink:href="#unlocked"></use></svg>

**Expected behavior**

Add the attributes aria-hidden="true" and focusable="false" to the SVG element to ensure it is not read or in the tab order.

<svg aria-hidden="true" focusable="false" width="20" height="20"><use href="#unlocked" xlink:href="#unlocked"></use></svg>

**Comment by Ckbhatia** [comment]
Is there anyone working on this issue?

**Comment by webron** [comment]
No. PRs welcome.

**Comment by mgifford** [comment]
Some guidance on SVG images [https://cariefisher.com/a11y-svg-updated/](https://cariefisher.com/a11y-svg-updated/)  
  
Still some language issues, but that might be able to be resolved with inline SVGs.
